src/actuator/ActuatorFAST.C

```diff
@@ -43,13 +43,19 @@
 #include <stdexcept>
 #include <cmath>
 
+// The utilities used for actuator
+#include "actuator/UtilitiesActuator.h"
+
 namespace sierra {
 namespace nalu {
 
 // constructor
 ActuatorFASTInfo::ActuatorFASTInfo() : ActuatorInfo()
 {
-  // nothing to do
+  // Initialize the filtered lifting line theory correciton to false
+  // This is later read from the yaml input file and can be changed
+  fllt_correction_ = false;
```

Comment:
Better to do this as either in the header or in the c'tor initializer list

Reply:
good catch